README.md

```diff
@@ -3,9 +3,9 @@
 
 # EEW Website
 
-A website for the [Environmental Enforcement Watch](//environmentalenforcementwatch.org) project by [EDGI](//envirodatagov.org)
+The environmentalenforcementwatch.org website for the [Environmental Enforcement Watch](//environmentalenforcementwatch.org) project by [EDGI](//envirodatagov.org)
 
-Currently staged at https://edgi-govdata-archiving.github.io/EEW-Website/
+Staging site at [staging.environmentalenforcementwatch.org](staging.environmentalenforcementwatch.org)
```

Comment:
Link needs `//` like above links. Currently tries to link to a github page with that path and 404s.

Reply:
ha, thanks for picking that up